src/components/USMap.jsx

```diff
@@ -6,8 +6,10 @@ import { StyleSheet } from 'react-look'
 import EventDetails from './EventDetails'
 import theme from '../theme'
 import MapZoomOut from './MapZoomOut'
+import HoveredPopup from './HoveredPopup'
 import { connect } from 'react-redux'
 import { selectState } from '../actions'
+import moment from 'moment'
```

Comment:
We need to add this to package.json and npm-shrinkwrap.  If you do `npm install --save moment` it should do that automatically.